libs/src/utils.ts

```diff
@@ -193,8 +193,12 @@ export function isUnique<T>(
 }
 
 export function parseIdentifier(identifier: string | null | undefined): string {
```

Comment:
@daywiss this function errored when you pass in a regular utf8 string. Caused some trouble in storybook. I figured we might as well just let the function still work when you pass in a utf8 string since that's what we want to end up with. let me know if that doesn't make sense